src/java.base/share/classes/java/util/Random.java

```diff
@@ -80,8 +81,143 @@
         name = "Random",
         i = 48, j = 0, k = 0,
         equidistribution = 0
-)
+        )
 public class Random implements RandomGenerator, java.io.Serializable {
+
+    /**
+     * Class used to wrap a {@link java.util.random.RandomGenerator} to
+     * {@link java.util.Random}.
+     */
+
+    @SuppressWarnings("serial")
+    private static class RandomWrapper extends Random implements RandomGenerator {
```

Comment:
Isn't ` implements RandomGenerator` redundant because `Random` already implements `RandomGenerator`?

Reply:
I prefered to keep the interface explicit to the reader